sdk/go/common/resource/alias.go

```diff
@@ -21,10 +22,6 @@ func (a *Alias) GetURN() URN {
 	return CreateURN(a.Name, a.Type, a.Parent, a.Project, a.Stack)
 }
 
-func (a *Alias) NoParent() bool {
-	return a.Parent == ""
-}
-
```

Comment:
Technically, this is a breaking change. However, I think it's worth making the change because it's unlikely anyone else is actually using this directly.

The alternative is to keep the `NoParent` method as-is (potentially marking it as deprecated) and add a new bool field for `NoParent` with some other name.

Reply:
I'm fine with this, most of the things in /common aren't really supposed to be used for public consumption.